src/validation.cpp

```diff
@@ -504,7 +505,7 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
                 {
                     for (const CTxIn &_txin : ptxConflicting->vin)
                     {
-                        if (_txin.nSequence < std::numeric_limits<unsigned int>::max()-1)
+                        if (_txin.nSequence <= MAX_BIP125_RBF_SEQUENCE)
```

Comment:
`MAX_BIP125_RBF_SEQUENCE` is `0xfffffffd`
The old value was `0xffffffff` (on supported platforms)
Is this intended?